site/training/training-for-model-validators.qmd

```diff
@@ -0,0 +1,359 @@
+- Robustness concerns
+- And more!
+
+Explore existing findings, <br>then click [{{< fa chevron-right >}}]() to continue.
+::::
+
+## {background-iframe="https://app.prod.validmind.ai/reports" background-interactive="true" data-preload="yes"}
+
+
+:::: {.fr .f4 .mv7 .nr5 .pa4 .bg-light-pink .ba .b--black-20 .bw1 .br3 .near-black .shadow-5}
+Under **Reports**, you can <br>find:
+
+- Executive summaries
+- Model tracking
+- Risk assessments
+- Compliance review
+- Remediation tracking
+
+Explore existing reports, <br>then click [{{< fa chevron-right >}}]() to continue.
+::::
+
+# 1. Review documentation
+
+## {.scrollable}
+
+:::: {.columns}
+::: {.column width="30%"  .pr4}
+Verify model documentation is complete and complies with regulatory standards by cross-checking against your guidelines. 
+:::
+
+::: {.column width="70%" .bl .pl4 .f3}
+### Review model documentation
+
+{{< include ../guide/_review-model-documentation.qmd >}}
+
+<form method="get" action="../guide/review-model-documentation.html" target="_blank">
+   <button type="submit" style="color: white; background-color: #de257e; border-radius: 8px; border: none; font-size: 24px; padding: 6.25px 12.5px; margin-top: 10px; margin-bottom: 20px;">Learn more ...</button>
+</form>
+:::
+::::
+
+Try it yourself on the next page. {{< fa hand-point-right >}}
+
+## {background-iframe="https://app.prod.validmind.ai" background-interactive="true" data-preload="yes"}
+
+::: {.absolute bottom=15 .w-100 .f4 .tc .pl4 .bg-light-pink .ba .b--black-20 .bw1 .br3 .near-black .shadow-5}
+**Try it:** Review the documentation for the **[Quickstart] Customer Churn Model**. | Click [{{< fa chevron-right >}}]() to continue.
+:::
+
+## {.scrollable}
+
+:::: {.columns}
+::: {.column width="30%"  .pr4}
+Have a question about the model? Collaborate with your developer right in the model documentation.
+
+<br>Try it yourself on the next page. {{< fa hand-point-right >}}
+:::
+
+::: {.column width="70%" .bl .pl4 .f3}
+::: {.f5}
+:::
+
+::: {.panel-tabset}
+
+{{< include ../guide/_collaborate-with-others-comments.qmd >}}
+
+:::
+
+<form method="get" action="../guide/collaborate-with-others.html" target="_blank">
+   <button type="submit" style="color: white; background-color: #de257e; border-radius: 8px; border: none; font-size: 24px; padding: 6.25px 12.5px; margin-top: 10px; margin-bottom: 20px;">Learn more ...</button>
+</form>
+:::
+::::
+
+## {background-iframe="https://app.prod.validmind.ai/" background-interactive="true" data-preload="yes"}
+
+::: {.absolute bottom=15 .w-100 .f4 .tc .pl4 .bg-light-pink .ba .b--black-20 .bw1 .br3 .near-black .shadow-5}
+**Try it:** Post a comment, reply to it, and then resolve the thread. Review the comment activity. | Click [{{< fa chevron-right >}}]() to continue.
+:::
+
+# 2. Analyze test results
+
+## {.scrollable}
+
+:::: {.columns}
+::: {.column width="30%"  .pr4}
+Locate the test results in the documentation, review the data, and identify issues with the model. 
```

Comment:
**GLOBAL COMMENT:** Same idea here about the sizing and colour, and moving the `Try it [...]` to it's own line beneath the two grid columns.

Reply:
> **GLOBAL COMMENT:** Same idea here about the sizing and colour, and moving the `Try it [...]` to it's own line beneath the two grid columns.

Will transfer your othher suggestion globally. The `Try it ...` text is in the left column only when there is otherwise not enough room underneath the two columns.